src/main/java/org/embl/mobie/io/ome/zarr/loaders/N5OMEZarrImageLoader.java

```diff
@@ -343,17 +357,32 @@ private double[] getXYZScale( OmeZarrMultiscales.CoordinateTransformations coord
         double[] xyzScale = null;
         if (scale != null && zarrAxesList != null) {
             int scalesFirstIndexBackward = scale.length - 1;
-            if (zarrAxes.containsXYZCoordinates()) {
-                xyzScale = new double[]{
-                        scale[scalesFirstIndexBackward],
-                        scale[scalesFirstIndexBackward - 1],
-                        scale[scalesFirstIndexBackward - 2] };
+            if (zarrAxes.hasChannels()) {
+                if (zarrAxes.is3DWithChannels() || zarrAxes.is2D()) {
```

Comment:
I don't think we need the `zarrAxes.is2D` here, since this will always have `zarrAxes.hasChannels(): false`